frontend/src/containers/SignInPage/index.tsx

```diff
@@ -0,0 +1,67 @@
+import * as React from 'react';
+import {useForm} from 'react-hook-form';
+import { yupResolver } from '@hookform/resolvers/yup';
+import * as yup from 'yup';
+// import { isValidEmail, isValidPassword } from 'healthcare-shared/helpers';
+
+import styles from './signin.module.scss';
+
+enum RegisterPayloadKey {
+  EMAIL = 'email',
+  PASSWORD = 'password',
+}
```

Comment:
You need for all application, so it would be better to move it to the `shared` folder